src/App.js

```diff
@@ -1,13 +1,53 @@
+import {
+  BrowserRouter as Router, Routes, Route, Link,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
-import './App.css';
 import Quote from './components/quote';
+import Home from './components/Home';
+import './App.css';
+
+function Header() {
+  return (
+    <header>
+      <h1>Math Magicians</h1>
+      <h2>Welcome to our page!</h2>
+    </header>
+  );
+}
 
 function App() {
   return (
-    <div>
-      <Calculator />
-      <Quote />
-    </div>
+    <Router>
+      <div className="firstcontainer">
+        <div className="navbar">
+          <nav>
+            <ul>
+              <li>
+                <Link to="/">Home</Link>
+              </li>
+              <li>
+                <Link to="/calculator">Calculator</Link>
+              </li>
+              <li>
+                <Link to="/quote">Quote</Link>
+              </li>
+            </ul>
+          </nav>
+        </div>
+        <Header />
+        <main>
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/calculator" element={<Calculator />} />
+            <Route path="/quote" element={<Quote />} />
```

Comment:
- [ ] [OPTIONAL] you can add styling to your quote page, you can add some padding to the page, give the paragraph width, centre your text etc. just make it look nice 
![Screenshot from 2023-03-28 20-14-29](https://user-images.githubusercontent.com/66097371/228344346-9822bd2a-e8a2-48ec-90ff-2fffb6d5c5ce.png)